profiler/profile.go

```diff
@@ -319,15 +319,18 @@ type pprofileDeltaProfiler struct {
 // Otherwise, deltas will be computed for every value.
 func newDeltaProfiler(cfg *config, v ...pprofutils.ValueType) deltaProfiler {
 	switch cfg.deltaMethod {
-	case "fastdelta":
-		return newFastDeltaProfiler(v...)
+	// TODO: slowdelta and comparing implementation can be removed after 2022-04-11.
+	case "slowdelta":
+		return &pprofileDeltaProfiler{delta: pprofutils.Delta{SampleTypes: v}}
 	case "comparing":
 		return newComparingDeltaProfiler(
 			cfg,
 			&pprofileDeltaProfiler{delta: pprofutils.Delta{SampleTypes: v}},
 			newFastDeltaProfiler(v...))
+	case "fastdelta":
+		fallthrough
```

Comment:
Is this needed? Can we just have the `default` case and no `case "fastdelta"` case?

Reply:
Sure 86527755adef77d71fe982ab3416c466536bcb6b 👍